lago/virt.py

```diff
@@ -219,6 +221,82 @@ def export_vms(self, vms_names, standalone, dst_dir, compress):
             for _vm in vms:
                 _vm.export_disks(standalone, dst_dir, compress)
 
+        self.generate_init(dst_dir, out_format)
+
+    def generate_init(self, dst_dir, out_format, filters=None):
+        """
+        Generate an init file which represents this env and can
+        be used with the images created by self.export_vms
+
+        Args:
+            dst_dir (str): path to the new init file
+            out_format (plugins.output.OutFormatPlugin):
+                formatter for the output
+            filters (list): list of paths to keys that should be removed from
+                the init file
+        Returns:
+            None
+        """
+        with LogTask('Exporting init file to: {}'.format(dst_dir)):
+            if not filters:
+                filters = [
+                    'domains/*/disks/*/metadata',
+                    'domains/*/metadata/deploy-scripts', 'domains/*/snapshots',
+                    'domains/*/name', 'nets/*/mapping'
+                ]
+            spec = self.get_env_spec(filters)
+
+            for _, domain in spec['domains'].viewitems():
+                for disk in domain['disks']:
+                    if disk['type'] == 'template':
+                        disk['template_type'] = 'qcow2'
```

Comment:
Not sure if this is the right place for this logic. Should probably be a different method.